exercises/protein-translation/canonical-data.json

```diff
@@ -118,13 +118,13 @@
           "expected": ["Methionine","Phenylalanine","Tryptophan"]
         },
         {
-          "description": "Translation stops if STOP codon present 1",
+          "description": "Translation stops if STOP codon present in short string",
```

Comment:
Perhaps `Translation stops if STOP codon at end of sequence` vs below `Translation stops if STOP codon in middle of sequence`?

Reply:
or
`Translation stops when STOP codon encountered`

Reply:
Still need to distinguish between this case and the next... perhaps `Translation stops when STOP codon encountered at end of sequence` vs `Translation stops when STOP codon encountered in middle of sequence`?